src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -1689,6 +1655,73 @@ class spell_dk_death_grip_initial : public SpellScriptLoader
     }
 };
 
+// 49560 - Death Grip (second part)
+class spell_dk_death_grip : public SpellScriptLoader
```

Comment:
while your are moving spellscripts, move all 3 spells to previous location to keep alphabetical order